README.md

```diff
@@ -46,6 +46,14 @@ Download the latest binary executable for your operating system:
   sudo tar xvzf tkn_0.3.1_Linux_arm64.tar.gz -C /usr/local/bin/ tkn
   ```
 
+  You can also use [@chmouel](https://github.com/chmouel)'s unofficial
+  copr package repository for the time being.
+
+  ```shell
+  dnf copr enable chmouel/tektoncd-cli
```

Comment:
Nit: Maybe a mention somewhere that this is for fedora?

Reply:
Thanks!